x-pack/solutions/observability/plugins/observability/test/scout/ui/fixtures/constants.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const BIGGER_TIMEOUT = 100000 as const;
```

Comment:
This is too much 😄 Our test timeout is 60 seconds. Is it really possible for a page to load that long?